lib/adagrams.rb

```diff
@@ -0,0 +1,109 @@
+# wave1 code
+def draw_letters
+  pool_of_letters = {
+    "A" => 9, "B" => 2, "C" => 2, "D" => 4, "E" => 12, "F" => 2, "G" => 3, "H" => 2, "I" => 9, "J" => 1, "K" => 1, "L" => 4, "M" => 2, "N" => 6, "O" => 8, "P" => 2, "Q" => 1, "R" => 6, "S" => 4, "T" => 6, "U" => 4, "V" => 2, "W" =>2, "X" => 1, "Y" => 2, "Z" => 1
+  }
+
+  players_letters = []
+  alphabet = ("A".."Z").to_a
+  
+  until players_letters.length == 10
```

Comment:
This is clever. Consider this other method to draw_letters that uses the pool_of_letters hash to create an array of letters. You could then shuffle this array of letters and pop off a letter 10 times.

```ruby
letters = []
pool_of_letters.each do |letter, count|
  count.times do
    letters << letter
  end
end
```